src/Lykke.BigDecimal/BigDecimal.Comparable.cs

```diff
@@ -0,0 +1,243 @@
+namespace System.Numerics
+{
+    public partial struct BigDecimal : IComparable, IComparable<BigDecimal>, IComparable<BigInteger>, IComparable<decimal>
+    {
+        /// <summary>
+        ///    Compares this instance to a specified object.
+        /// </summary>
+        /// <param name="obj">
+        ///    The object to compare.
+        /// </param>
+        /// <returns>
+        ///    Returns an integer that indicates whether the value of this instance is less than, equal to, or greater than the value of the specified object.
```

Comment:
I'd like to see something more explicit in comment. Smth like "Returns -1 if ..."